### PR TITLE
Update README file with some examples on dev env with proxying

### PR DESCRIPTION
**Pull request by ddonahue007**
Updated readme file for dev experience

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -53,6 +53,48 @@ There is a [live editor](https://data-driven-forms.org/live-editor) where you ca
...
+  
+Example: 
+``` javascript
+  const { config: webpackConfig, plugins } = config({
+    rootFolder: resolve(__dirname, '../'),
+    debug: true,
+    useFileHash: false,
+    deployment: process.env.BETA ? 'beta/apps' : 'apps',
+    useProxy: true,
+    appUrl: `/beta/settings/applications`,
```
Comment (rvsia) [inline README.md:70]: `appUrl: process.env.BETA ? '/beta/settings/applications' : '/settings/applications'`
  
  Can you please change it to this?

Comment (ddonahue007) [reply README.md]: Done

Comment (ddonahue007) [reply README.md]: Would you like to see this in both examples?

**Review by rvsia — APPROVED** [review]

**Comment by rvsia** [comment]
cc @karelhala @Hyperkid123

**Review by karelhala — APPROVED** [review]

**Pull request merged**